picard/const/__init__.py

```diff
@@ -163,3 +163,11 @@
 )
 
 SCRIPT_LANGUAGE_VERSION = '1.1'
+
+BROWSER_INTEGRATION_IPV6 = False
```

Comment:
If we want to provide this as an experimental feature we could have it controlled by an environment variable or such.

Reply:
I don't think that's necessary at this point, I removed IPv6 specific code anyway (it was more a proof of concept than anything).